commerce_gdpr.admin.inc

```diff
@@ -0,0 +1,139 @@
+<?php
+
+/**
+ * @file
+ * Contains admin forms for the module.
+ */
+
+/**
+ * Configuration form builder.
+ */
+function commerce_gdpr_admin_form($form, &$form_state) {
+
+  // Issue a warining if a site-specific hashing salt is not set.
+  if (!variable_get('commerce_gdpr_salt', FALSE) && empty($form_state['input'])) {
+    drupal_set_message(t('To ensure increased safety of anonymized data, please set a platform-specific hashing salt. More information can be found on the module !help.', array(
+      '!help' => l(t('help page'), 'admin/help/commerce_gdpr', array(
+        'attributes' => array('target' => '_blank'),
+      )),
+    )), 'warning');
+  }
+
+  $form['data_retention'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Order data retention period'),
+    '#default_value' => variable_get('commerce_gdpr_data_retention', 0),
+    '#description' => t('Enter number of days after which inactive data will be anonymized, 0 - no automatic anonymization.'),
+  );
+
+  $form['direct_processing'] = array(
+    '#type' => 'checkbox',
+    '#title' => t('Process items immediately'),
+    '#description' => t('Process items immediately on the user form and during the "People" page bulk operation rather then enqueing the operations. NOTE: may cause performance issues in case of users with many orders.'),
+    '#default_value' => variable_get('commerce_gdpr_direct_processing', 0),
+  );
+
+  $form['user_button_text'] = array(
+    '#type' => 'textfield',
+    '#title' => t('User account anonymization button text'),
+    '#default_value' => variable_get('commerce_gdpr_user_button_text', 'I want to be forgotten'),
```

Comment:
lack of t()

Reply:
It's a default variable value, I thought it should be constant :)